views/administration/forms/form-invitation-student.hbs

```diff
@@ -1,10 +1,10 @@
 <input name="schoolId" type="hidden" data-force-value="true" value="{{currentSchool}}" />
 <div class="form-group">
-    <label for="invitation-link">Um in die HPI Schul-Cloud einzuladen, verteile folgenden Link an: </label>
+    <label for="invitation-link">{{$t "administration.label.toInviteToTheSchoolCloud" }} </label>
     <p>
-        <b>Schüler unter {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahre:</b> die Eltern <br/>
-        <b>Schüler ab {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahre:</b> an die Schüler selbst <br/>
+        <b>{{$t "administration.global.text.studentUnderYears" (dict "age" CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS)}} </b> {{$t "administration.form_InvitationStudent.text.theParents" }} <br/>
+        <b>{{$t "administration.form_InvitationStudent.text.studentsFromYears" (dict "age" CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS) }} </b> {{$t "administration.form_InvitationStudent.text.toTheStudentsThemselves" }} <br/>
```

Comment:
this would be good to put in one key as well
{{{ }}} to avoid html stripping

Reply:
done ;)